backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/api/IDiagramContext.java

```diff
@@ -67,12 +68,12 @@ public void reset() {
 
         @Override
         public List<ViewCreationRequest> getViewCreationRequests() {
-            return List.of();
+            return new ArrayList<>();
```

Comment:
We do not want to depend on anything else in our `NoOp` classes. The choice of `List.of()` is done on purpose. It may not match what's needed for some tests but then it should be customized in said tests.

Reply:
Done